cmd/add.go

```diff
@@ -46,6 +48,7 @@ func (ac *AddCommand) runAdd(cmd *cobra.Command, args []string) error {
 	file, _ := ac.command.Flags().GetString("file")
 	cover, _ := ac.command.Flags().GetBool("cover")
 	contextName, _ := ac.command.Flags().GetString("context-name")
+	selectContext, _ := ac.command.Flags().GetBool("select-context")
```

Comment:
Perhaps `GetString` is used here and followed by the relevant checks: exists, does not exist, is empty, etc.
This would allow for better automated testing in e2e test.

```go
selectContext, _ := ac.command.Flags().GetString("select-context")
```

Reply:
I want to choose contexts interactively, so I use **bool** type.
```bash
kubecm add -f kubeconfig1 --select-context
```
For my understanding, you want this:
```bash
kubecm add -f kubeconfig1 --select-context context1
```
It seems like we couldn't implement different types for the same flag (**bool** and **string**) in Cobra. So I think your idea should be implemented by a different flag.

Reply:
Indeed, a new flag should be created here.

Reply:
@cr7258 But this new flag is a new feature, which is not part of this PR, would you like to create a new PR to contribute this this feature? I will merge this PR first.

Reply:
@sunny0826 I will contribute this feature in a new PR.

Reply:
extremely grateful !